torchvision/io/_video_opt.py

```diff
@@ -306,13 +306,13 @@ def _read_video_from_memory(
     video_height=0,  # type: int
     video_min_dimension=0,  # type: int
     video_max_dimension=0,  # type: int
-    video_pts_range=(0, -1),  # type: List[int]
+    video_pts_range=None,  # type: Optional[List[int]]
```

Comment:
This was probably done to avoid a mutable default argument. Are we fixed on `List`s and `Tuple`s or can we use `Sequence`? If not, I don't see a better way than mine to also pass type checks.

Reply:
We are fixed with `List` because we need this to work with torchscript.

Reply:
The way you are doing it is good